frontend/locales/cs-CZ.json

```diff
@@ -203,8 +203,8 @@
         "email": "Email",
         "follow_dev": "Sledovat vývojáře",
         "footer": {
-            "api_link": "'<a href=\"https://homebox.software/en/api/\" target=\"_blank\">'API'</a>'",
-            "version_link": "'<a href=\"https://github.com/sysadminsmedia/homebox/releases/tag/v'{ version }\" target=\"_blank\"> Verze: { version } Sestavení: { build } '</a>'"
+            "api_link": "<a href=\"https://homebox.software/en/api/\" target=\"_blank\">API</a>",
+            "version_link": "<a href=\"https://github.com/sysadminsmedia/homebox/releases/tag/{ version }\" target=\"_blank\">Verze: { version } Sestavení: { build }</a>"
```

Comment:
There must be the escape `'` characters around the XML parts otherwise VueJS will crash the frontend.